### ISSUE TITLE
Can I pass options in simple_table_for that can be used by the report?

### ISSUE THREAD

**Issue by awh-tokyo** [bobbytables/table_cloth#46]
The documentation specifies that we can conditionally display columns like so:

  column :email, if: :admin?

Where "admin?" is defined in the report class.

What I'd like to do is pass some report options through this line:

<%= simple_table_for @users, with: UserTable %>

, those options being visible by the report class's methods.

My application has a few different ways of listing the same data, so it makes sense that I should have a single table definition and then just conditionally display the appropriate columns.

**Comment by bobbytables** [comment]
Yeah, this has been a bottleneck for me in the past as well. I'm not sure how to currently implement it within the design choices of table cloth.

Also, sorry for this delayed response. Slipped by.

**Comment by awh-tokyo** [comment]
FYI, for anybody else who has the same issue, this is what I ended up doing.  It's not beautiful, but it works:

  <% @report_type = "multi_history" %>
  <%= simple_table_for @inventory_histories, with: InventoryHistoryTable %>

then, in the table definition:

  def show_vendor?
    view.instance_variable_get(:@report_type).eql?("multi_history")
  end

  def show_actors?
    view.instance_variable_get(:@report_type).eql?("part_history")
  end
